app/src/main/res/xml/pref_general.xml

```diff
@@ -78,12 +78,14 @@
             android:key="@string/key_skin"
             android:title="@string/skin" />
 
-        <SwitchPreference
-            android:defaultValue="true"
-            android:key="@string/key_use_darkmode"
+        <ListPreference
+            android:defaultValue="system"
```

Comment:
Default to dark to not surprise existing users
```suggestion
            android:defaultValue="dark"
```

Reply:
Maybe you can work on it and make a new pull request. so we can finish it and hopefully MilosKozak will merge it in the dev brnach in the next time

Reply:
If it's working for you, as it also worked for me before my changes, then I'm fine with leaving it and letting someone else also test to be sure.